week1/index_products.py

```diff
@@ -103,13 +107,21 @@ def index_file(file, index_name):
             xpath_expr = mappings[idx]
             key = mappings[idx + 1]
             doc[key] = child.xpath(xpath_expr)
-        #print(doc)
+        logger.info(doc)
         if 'productId' not in doc or len(doc['productId']) == 0:
             continue
         #### Step 2.b: Create a valid OpenSearch Doc and bulk index 2000 docs at a time
-        the_doc = None
+        the_doc = { "_index": index_name, "_id": doc["sku"][0], "_source": doc }
+        docs_indexed += 1
```

Comment:
nitpick: the document hasn't been indexed yet so you might end up reporting incorrectly